src/pkg/cli/getServices.go

```diff
@@ -55,10 +56,11 @@ func GetServices(ctx context.Context, projectName string, provider client.Provid
 	for i, si := range servicesResponse.Services {
 		printServices[i] = printService{
 			Service:     si.Service.Name,
+			ID:          si.Etag,
 			ServiceInfo: si,
 		}
 		servicesResponse.Services[i] = nil
 	}
 
-	return term.Table(printServices, []string{"Service", "Etag", "PublicFqdn", "PrivateFqdn", "Status"})
+	return term.Table(printServices, []string{"Service", "ID", "PublicFqdn", "PrivateFqdn", "Status"})
```

Comment:
Make sure `Status` is the one we want and not `State`. `State` might be more useful, because it would show things like "out of memory".